packages/docs/docs/careplans/tasks/tasks.md

```diff
@@ -0,0 +1,140 @@
+
+:::caution
+
+While this functionality is powerful, it can be complex to maintain and operationalize. Medplum recommends that most implementations start with a single-level [`Task`](/docs/api/fhir/resources/task) hierarchy, and gradually add depth over time.
+
+:::
+
+## Examples
+
+| Use Case                                        | Task Owner                    | Focal Resource       | Example `businessStatuses`                                                                                                                                                                         | Additional Info                                                  |
+| ----------------------------------------------- | ----------------------------- | -------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ---------------------------------------------------------------- |
+| Complete patient intake questionnaire           | New Patient. Care Coordinator | `Questionnaire`      | <ol><li>Questionnaire sent to patient</li><li>First reminder sent</li><li>Second reminder sent</li><li>Questionnaire completed by patient</li><li>Responses reviewed by care coordinator</li></ol> | Use `Task.output` to reference resulting `QuestionnaireResponse` |
+| Review lab report                               | Physician                     | `DiagnosticReport`   | <ol><li>Report Available</li><li>Assigned to physician</li><li>Reviewed by physician</li><li>Discussed with patient</li></ol>                                                                      |                                                                  |
+| Verify patient identity (e.g. driver's license) | Patient. Care Coordinator     | `DocumentReference`  | <ol><li>Identification document requested</li><li>Documentation received</li><li>Documentation received</li><li>Documentation verified</li></ol>                                                   | See also:                                                        |
+| Complete encounter notes                        | Physician                     | `ClinicalImpression` | <ol><li>Encounter complete. Physician note required</li><li>Note drafted</li> <li>Note finalized</li></ol>                                                                                         | Use `Task.encounter` to reference the original encounter         |
+
+## See Also
+
+- The [FHIR Workflow Specification](http://hl7.org/fhir/R4/workflow.html)
+- [Medplum Task Demo](https://github.com/medplum/medplum-task-demo)
```

Comment:
More links: 
- https://www.medplum.com/blog/task-management-apps#dashboards
- https://www.medplum.com/docs/search/basic-search#not

Also - add a link from https://www.medplum.com/docs/charting#data-model to tasks.md

Reply:
Added, though I think the link to `:not` is misplaced. For unassigned tasks, `:missing` is the correct search